Taskfile.yml

```diff
@@ -8,7 +8,11 @@ vars:
   G_BUILD_DIR: "{{.ROOT_DIR}}/build"
   G_CLP_FFI_JS_BUILD_DIR: "{{.G_BUILD_DIR}}/clp-ffi-js"
   G_CLP_FFI_JS_CHECKSUM: "{{.G_BUILD_DIR}}/clp-ffi-js.md5"
-  G_CLP_FFI_JS_TARGET_NAME: "ClpFfiJs"
+  G_CLP_FFI_JS_ENV_NAMES: "node;worker"
```

Comment:
This is not a big deal, but for clarity, I would rather make this a map and then assemble the `;`-delimited string in `config-cmake-project`.

Reply:
I agree. Let's see if the new approach makes more sense: 6df29f396f2930ea45d80b81dcf40fb9c684f3d3